insights/login.py

```diff
@@ -0,0 +1,22 @@
+import ConfigParser
+import selenium_utility
+from locators import *
+
+locator = LoginPageLocators()
+config = ConfigParser.RawConfigParser()
+config.read('pool.conf')
+
+
+class Login_test(selenium_utility.Base):
```

Comment:
check class name naming conventions -> https://www.python.org/dev/peps/pep-0008/#class-names 
There is no test in that class, why the name *test in class name?